src/ec/packages/ec-component-accordion2/accordion2.story.js

```diff
@@ -3,32 +3,39 @@ import { storiesOf } from '@storybook/html';
 import { withKnobs, text } from '@storybook/addon-knobs';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
 import withCode from '@ecl-twig/storybook-addon-code';
-
+import { buttonLabels, getExtraKnobs } from '@ecl-twig/story-utils';
 import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
 import demoData from './demo/data';
 
 import accordion2 from './ecl-accordion2.html.twig';
 import notes from './README.md';
 
+const PrepareAccordion2 = data => {
+  data.items.forEach((item, index) => {
+    const { content, toggle } = item;
+
+    item.toggle.label = text(
+      `toggle.label ${index}`,
+      toggle.label,
+      buttonLabels.required
+    );
+
+    item.content = text(`content ${index}`, content, buttonLabels.required);
+    item.toggle.icon.path = defaultSprite;
```

Comment:
this should be a knob, everything should be a knob, not sure what can still be unclear about what we are doing in this "demo" track but i'll try again: we are trying to expose EVERY thing you can do with a component, every single property you can or you have to set, giving all the possibilities to customise the demo of the component and then reporting the correct code for the include in the notes tab.
if you have to set a a path to an icon to make an icon work i want that to be a knob, (a select knob with only that option) and this is valid for all the rest, there are now plenty of examples of this being done in multiple components.